src/components/hint/index.js

```diff
@@ -358,7 +358,7 @@ class Hint extends BaseComponent {
             this.getHintPadding()
           ]}
           pointerEvents="box-none"
-          testID={testID}
+          testID={`${testID}.animatableView`}
         >
```

Comment:
Don't change this. 
There's always a main element that gets the original testID that the user pass. 
Also, changing this is a breaking change cause it will break tests that depend on that.